content/docs/started/getting-started-gke.md

```diff
@@ -84,8 +84,9 @@ Run the following steps to deploy Kubeflow:
     cd ${KUBEFLOW_REPO}
     export KUBEFLOW_TAG={{% kf-stable-tag %}}
     curl https://raw.githubusercontent.com/kubeflow/kubeflow/${KUBEFLOW_TAG}/scripts/download.sh | bash
+    cd ..
      ```
-   * **KUBEFLOW_REPO** directory where you want to download the source to
+   * **KUBEFLOW_REPO** full path to the directory where you want to download the source to
```

Comment:
Can we change this environment variable KUBEFLOW_SRC?

Otherwise it interferes with kfctl.sh which uses KUBEFLOW_REPO?

Reply:
I think the problem here is that we are using environment variable KUBEFLOW_REPO but we should be using a different variable like KUBEFLOW_SRC.

The commands here shouldn't depend on KUBEFLOW_REPO being a full path; i.e. mkdir and cd work with the whatever directory you specify.

download.sh will then download into the current directory.

KUEFLOW_REPO is used by kfctl.sh. It will automatically be set based on the location of kctl.sh.
The environment variable KUBEFLOW_REPO should only be set if you want to override the default for some reason.

Reply:
Note that PR [#277](https://github.com/kubeflow/website/pull/277) changes KUBEFLOW_REPO to KUBEFLOW_SRC in this and other files.